### PR TITLE
python3Packages.identify: 1.6.1 -> 2.2.10

### PR DESCRIPTION
**Pull request by austinbutler**
###### Motivation for this change

Update package.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by austinbutler** [comment]
Result of `nixpkgs-review pr 128938` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>7 packages built:</summary>
  <ul>
    <li>pre-commit (python38Packages.pre-commit)</li>
    <li>python38Packages.editdistance-s</li>
    <li>python38Packages.identify</li>
    <li>python39Packages.editdistance-s</li>
    <li>python39Packages.identify</li>
    <li>python39Packages.pre-commit</li>
    <li>wlc</li>
  </ul>
</details>

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 128938` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>7 packages built:</summary>
  <ul>
    <li>pre-commit (python38Packages.pre-commit)</li>
    <li>python38Packages.editdistance-s</li>
    <li>python38Packages.identify</li>
    <li>python39Packages.editdistance-s</li>
    <li>python39Packages.identify</li>
    <li>python39Packages.pre-commit</li>
    <li>wlc</li>
  </ul>
</details>

**Review by fabaff — APPROVED** [review]
Looks good to me.

**Comment by fabaff** [comment]
@ofborg eval

**Comment by cole-h** [comment]
@ofborg eval

(Sorry, ofborg's certs recently had an issue, meaning that it was unable to
react to PRs. This has been fixed, but all PRs filed in this period will need
ofborg to be manually kicked off.)

**Comment by risicle** [comment]
`nixpkgs-review` reveals no new failures, macos 10.15.

File: pkgs/development/python-modules/editdistance-s/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+, pytestCheckHook
+, cffi
+}:
+
+buildPythonPackage rec {
+  pname = "editdistance-s";
+  version = "1.0.0";
+
+
+  src = fetchFromGitHub {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/editdistance-s/default.nix:13]:
```suggestion
  version = "1.0.0";

  src = fetchFromGitHub {
```

**Pull request merged**